Cargo.toml

```diff
@@ -15,4 +22,7 @@ actix-web = { version = "4.4.0", features = ["rustls-0_21"]}
 rustls = "0.21.7"
 rustls-pemfile = "1.0.3"
 actix-cors = "0.6.4"
+reqwest = { version = "0.11", features = ["json"] }
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```

Comment:
I suggest to check if we need both features, especially `rt-multi-thread`
Also from what I found, there is an alternative crate, called `tokio-test`

Reply:
I removed the in commit [e8b1589a83bbd1d77dd250e34180fb33c02241d2](https://github.com/calendar-team/calendar-backend/pull/11/commits/e8b1589a83bbd1d77dd250e34180fb33c02241d2#diff-117f5e52f7246f4eff31874f46a65d966e1bab790eacb121dd7e857a33c4974f)
Nice catch

`tokio-test` doesn't have an alternative for the spawn function so we can't use it easily as an alternative